### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by mjalbert99**

### CODE REVIEW

**Comment by mjalbert99** [comment]
Added the calls for a user to log in to their account. Removed the check password function as it was outdated. Will start  main activity when successful

**Comment by mjalbert99** [comment]
Added option to nav menu to log out and added function to main activity to log out

**Comment by mjalbert99** [comment]
Finished the final code to log in and log out. Just need to test and make the login page the start-up activity

**Comment by mjalbert99** [comment]
If the user logs in and closes the app. When they reopen it they will be directed to the login page. If they sign out of the app they will be directed to the login page. Added sign out button to settings page based on Figma

**Comment by mjalbert99** [comment]
Added a basic test

**Comment by mjalbert99** [comment]
Final test implemented now ready for code review

File: app/src/main/java/com/example/activeamigo/LoginActivity.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                });
+            }
+        });
+
+        // Launch Register Page
+        findViewById(R.id.buttonRegister).setOnClickListener(view -> {
+            Intent intent = new Intent(LoginActivity.this, CreateAccountActivity.class);
+            startActivity(intent);
+        });
+
```
Comment (cknguyen123) [inline app/src/main/java/com/example/activeamigo/LoginActivity.java:70]: remove whitespace here... Consistency

**Review by cknguyen123 — COMMENTED** [review]
good

**Pull request merged**